src/lib/updateGrantsRecipients.js

```diff
@@ -137,6 +137,27 @@ async function getProgramPersonnel(grantId, programId, program) {
   return programPersonnelArray;
 }
 
+export const updateCDIGrantsWithOldGrantData = async (grantsToUpdate) => {
+  try {
+    const updatePromises = grantsToUpdate.map(async (grant) => {
+      if (grant.oldGrantId) {
+        const oldGrant = await Grant.findByPk(grant.oldGrantId);
```

Comment:
Why do the find and update in two separate calls to the db. It's good that all the update promises are awaiting together, but it could be even more efficient if each operation had a single db call that could be awaited together at the end

Reply:
Yes, it's true that there is a way to perform one trip to the database and slightly optimize performance. I chose this solution, though, since it has the advantage of being straightforward and readable. The performance optimization would be more relevant if there were many grants needing an update. As it stands, between 3/17/21 and 10/16/23, there are only about 110 grants with region 13. That makes it roughly 3.55 grants per month.